<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAction</class>
 <widget class="QWidget" name="UserAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>901</width>
     <height>711</height>
    </rect>
   </property>
   <property name="title">
    <string>EEG图像</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>通信连接状态：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>158</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>671</width>
      <height>651</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>EEG原始图</string>
     </property>
     <widget class="QCustomPlot" name="customplot_2" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>601</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>功率谱图</string>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customplot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>601</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
